app/src/main/java/com/example/sharingang/MapActivity.kt

```diff
@@ -0,0 +1,68 @@
+package com.example.sharingang
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.location.Location
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.databinding.DataBindingUtil
+import com.example.sharingang.databinding.ActivityMapBinding
+import com.google.android.gms.location.*
+import com.google.android.gms.tasks.CancellationTokenSource
+import com.google.android.gms.tasks.Task
+
+const val REQUEST_LOCATION:Int = 0
+class MapActivity : AppCompatActivity(){
```

Comment:
Maybe we should write a little doc comment for this activity, explaining what it does briefly